### ISSUE TITLE
Consider Grid--fit rename/Grid--distribute

### ISSUE THREAD

**Issue by timkelty** [suitcss/components-grid#51]
`Grid--fit`'s name can be a bit misleading, particularly because of the name of `u-sizeFit`.

One might think `Grid--fit` might be equivalent to:

```
    <div class="Grid">
      <div class="Grid-cell u-sizeFit">
        <div class="fixture-Box">1</div>
      </div>
      <div class="Grid-cell u-sizeFit">
        <div class="fixture-Box">2</div>
      </div>
      <div class="Grid-cell u-sizeFit">
        <div class="fixture-Box">3</div>
      </div>
    </div>
```

But it isn't.
`Grid--fit` _distributes_ cell widths to occupy an entire row equally.

I'm thinking maybe `Grid--distribute` would be more appropriate? If we wanted to support a "shirinkwrap" grid like the above example, we could repurpose `Grid--fit` for that.

Otherwise maybe `Grid--collapse`, or just dropping it all together. It's basically if you were using a standard flexbox, but still wanted your grid gutters (think horizontal list).

Thoughts @simonsmith @giuseppeg?

**Comment by simonsmith** [comment]
Agreed, that's not the best word. Would it be more like `u-sizeFill`?

**Comment by timkelty** [comment]
@simonsmith correct, the current version of Grid--fit is equivalent to putting u-sizeFill on every cell.

So maybe Grid--fill, Grid--fit?

**Comment by simonsmith** [comment]
If we chose `Grid-fill` that would keep it inline with the size utilities.

**Comment by simonsmith** [comment]
Actually what if we made that the default behaviour of `Grid`?

**Comment by timkelty** [comment]
Yeah - I think full width cells are still good for encouraging a mobile-first composition.

**Comment by timkelty** [comment]
> I'm thinking maybe Grid--distribute would be more appropriate? If we wanted to support a "shirinkwrap" grid like the above example, we could repurpose Grid--fit for that.

@simonsmith Do you think we should also repurpose Grid-fit as described?

**Comment by simonsmith** [comment]
@timkelty Yes! Totally forgot about that suggestion. Will make another PR

**Comment by timkelty** [comment]
@simonsmith I can do this today if you haven't started!

**Comment by simonsmith** [comment]
@timkelty By all means, jump on it

**Comment by simonsmith** [comment]
Fixed in #53
